From 58c0450c2c2343b79d78aa29ada05bb06202b7ae Mon Sep 17 00:00:00 2001
From: Andrew Hopkins <andhop@amazon.com>
Date: Fri, 10 Mar 2023 10:32:10 -0800
Subject: [PATCH] AWS-LC does not provide Time Stamp Authority functionality,
 define OPENSSL_NO_TS to notify code compiling with AWS-LC it is not provided
 (#864)

---
 include/openssl/opensslconf.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/openssl/opensslconf.h b/include/openssl/opensslconf.h
index 62bfb41a8..9879746e1 100644
--- a/include/openssl/opensslconf.h
+++ b/include/openssl/opensslconf.h
@@ -65,6 +65,7 @@
 #define OPENSSL_NO_SSL3_METHOD
 #define OPENSSL_NO_STATIC_ENGINE
 #define OPENSSL_NO_STORE
+#define OPENSSL_NO_TS
 #define OPENSSL_NO_WHIRLPOOL
 
 #endif  // OPENSSL_HEADER_OPENSSLCONF_H
